qemu/tests/cfg/dump_guest_core.cfg

```diff
@@ -3,7 +3,7 @@
     only x86_64 ppc64 ppc64le
     virt_test_type = qemu
     pre_command = "echo "<domain>        <type>  <item>  <value>" >> /etc/security/limits.conf"
-    pre_command += "echo "@root          soft    core    unlimited" >> /etc/security/limits.conf"
```

Comment:
These will add 2 lines each time you run the case. Suggest backup the original one or use `sed`.

Reply:
Yes,it is expected.
Okay,will backup it first and restore it after the test.